ejb-txn-remote-call/README.adoc

```diff
@@ -0,0 +1,1092 @@
+at https://github.com/wildfly/wildfly-operator/blob/master/deploy/operator.yaml[WildFly Operator Github repository].
+
+For the deployment to work correctly a https://github.com/wildfly/wildfly-operator/blob/master/deploy/service_account.yaml[service account],
+https://github.com/wildfly/wildfly-operator/blob/master/deploy/role.yaml[a role] and
+https://github.com/wildfly/wildfly-operator/blob/master/deploy/role_binding.yaml[a role binding] have to be created
+in the Kubernetes cluster.
+
+The follow-up step is creation of https://github.com/wildfly/wildfly-operator/blob/master/deploy/crds/wildfly_v1alpha1_wildflyserver_crd.yaml[`CustomResourceDefinition`]
+(abbreviated as *CRD*) which defines what capabilities the Operator provides and which things may be configured for the `WildFlyServer` `CustomerResource`.
+
+[NOTE]
+====
+If you clone the https://github.com/wildfly/wildfly-operator[WildFly Operator GitHub repository] to your
+local disk you may use the prepared script https://github.com/wildfly/wildfly-operator/blob/master/build/run-minikube.sh[build/run-minikube.sh]
+for that purpose.
+
+[source,sh,options="nowrap"]
+----
+git clone https://github.com/wildfly/wildfly-operator
+./wildfly-operator/build/run-minikube.sh
+----
+====
+
+The quickstart uses clustering.
+The {productName} clustering works with https://github.com/jgroups-extras/jgroups-kubernetes[jgroups `KUBE_PING`]
+protocol. This protocol requires having permission to list all available pods in scope of the `namespace`.
+The `default` `ServiceAccount` does not have such permissions.
+For development purposes it is possible to use
+https://github.com/wildfly/wildfly-operator/blob/master/examples/clustering/crds/role_binding.yaml[`RoleBinding` definition from WildFly Operator repository].
+The definition permits for the deployments to view details information about any Kubernetes object
+inside of the current `namespace`.
+
+[source,sh,options="nowrap"]
+----
+kubectl create -f https://raw.githubusercontent.com/wildfly/wildfly-operator/master/examples/clustering/crds/role_binding.yaml
+----
+
+When all this is set up, and the {productName} Operator `Pod` is running we may prepare a definition
+of the `CustomerResource` which makes the application deployed.
+The `CustomerResource` definition points to the built images `wildfly-quickstarts/client` and `wildfly-quickstarts/server`
+which has to be provided to Kubernetes.
+
+The WildFly Operator should now be running and the images will be available for use.
+For checking that the WildFly Operator is running check with the command `kubectl get pods` which
+should be showing the similar output to the following.
+
+[source,sh,options="nowrap"]
+----
+kubectl get pods
+
+NAME                                READY   STATUS    RESTARTS   AGE
+postgresql-7dd74c84dc-s8sh8         1/1     Running   0          2m44s
+wildfly-operator-5d4b7cc868-8qhjn   1/1     Running   0          16m
+----
+
+[#running_on_kubernetes_deploy_customer_resource]
```

Comment:
Is there supposed to be a heading here?  I assume the ID is here to create an anchor.  Recommended practice in technical documentation is to link to an anchor.

Reply:
In my original comment, I meant to say that recommended practice is to link to a heading. It would be better to have a heading, but I can live with this.

Reply:
I see. I added a new title here.